src/unix/notbsd/mod.rs

```diff
@@ -422,9 +426,28 @@ pub const CBAUD: ::tcflag_t = 0o0010017;
 
 pub const EFD_CLOEXEC: ::c_int = 0x80000;
 
+#[cfg(not(all(target_env = "musl",
+              any(target_arch = "x86",
+                  target_arch = "arm",
+                  target_arch = "asmjs"))))]
 pub const F_SETLK: ::c_int = 6;
+#[cfg(not(all(target_env = "musl",
+              any(target_arch = "x86",
+                  target_arch = "arm",
+                  target_arch = "asmjs"))))]
 pub const F_SETLKW: ::c_int = 7;
 
+#[cfg(all(target_env = "musl",
+          any(target_arch = "x86",
+              target_arch = "arm",
+              target_arch = "asmjs")))]
+pub const F_SETLK: ::c_int = 13;
+#[cfg(all(target_env = "musl",
+          any(target_arch = "x86",
+              target_arch = "arm",
+              target_arch = "asmjs")))]
+pub const F_SETLKW: ::c_int = 14;
```

Comment:
Could these `#[cfg]` annotations also be pushed down into the respective modules?